Mathlib/Tactic/Simpa.lean

```diff
@@ -25,8 +25,10 @@ This is a "finishing" tactic modification of `simp`. It has two forms.
   the `assumption` tactic.
 
 #TODO: implement `with ⋯` behavior
+#TODO: implement `!`
+#TODO: implement `?`
 -/
-elab "simpa " cfg?:(config)? disch?:(discharger)?
+elab "simpa" unfold:"!"? squeeze:"?"? cfg?:(config)? disch?:(discharger)?
```

Comment:
This doesn't work. `simpa!` will not parse the way you want, which is why there is the combinatorial stuff.